artemis-server/src/main/java/org/apache/activemq/artemis/core/server/impl/ActiveMQServerImpl.java

```diff
@@ -946,7 +946,7 @@ public boolean isReplicaSync() {
       } else if (activation instanceof SharedNothingBackupActivation) {
          return ((SharedNothingBackupActivation) activation).isRemoteBackupUpToDate();
       } else {
-         throw ActiveMQMessageBundle.BUNDLE.methodNotApplicable();
```

Comment:
Isn't this the only place where this error message was generated? If so, then it may be also deleted at

https://github.com/apache/activemq-artemis/blob/e5ef7d37ae255b9ed478bc4b45586cf8a179b18b/artemis-server/src/main/java/org/apache/activemq/artemis/core/server/ActiveMQMessageBundle.java#L383-L384

Reply:
@jdanekrh, yes. That is the only place where that message was used but I elected not to delete that method so that developers would know that the message's id (i.e. `119118`) shouldn't be used again since message ids should be unique.

Reply:
Makes sense, although ARTEMIS-1018 Duplicate error ids on different error mesages. Thanks for not making it worse, I guess.